pageserver/ctl/src/main.rs

```diff
@@ -90,19 +91,20 @@ async fn main() -> anyhow::Result<()> {
             layer_map_analyzer::main(&cmd).await?;
         }
         Commands::PrintLayerFile(cmd) => {
-            if let Err(e) = read_pg_control_file(&cmd.path) {
+            let cmd_path = Utf8PathBuf::from_path_buf(cmd.path).expect("non-Unicode path");
```

Comment:
I wonder for pagectl if the camino's Path could be used as part of those subcommand arguments already, to do less conversions down the line?